packages/mux-player/src/themes/classic/index.ts

```diff
@@ -0,0 +1,18 @@
+// @ts-ignore
+import theme from './theme.html';
```

Comment:
this probably should point at `./classic.html`?

Reply:
Oops yup, got lost in renaming lots of stuff. But now I'm worried that my local running dev example page didn't show any issues 🤔...